runtime/onert/core/include/backend/ITensor.h

```diff
@@ -103,6 +103,17 @@ class ITensor
    * @note  Higer dimension will be placed on front.
    */
   virtual ir::Shape getShape() const;
+
+  virtual bool is_subtensor() const { return false; }
+  virtual bool needMemoryMap() const { return false; }
+  virtual void enqueueWriteBuffer(const void *, bool)
+  {
+    throw std::runtime_error("This backend does not support enqueueWriteBuffer");
+  }
+  virtual void enqueueReadBuffer(void *, bool)
+  {
+    throw std::runtime_error("This backend does not support enqueueReadBuffer");
+  }
```

Comment:
Frankly I'm not a big fan of these introduced, but I also admit that there is another way that can be done quickly. We may accept this and we'll do some refactoring later.